src/libs/geotagging.c

```diff
@@ -1479,10 +1480,22 @@ static GtkWidget *_gui_init_datetime(dt_lib_datetime_t *dt, const int type, dt_l
         gtk_widget_set_sensitive(dt->widget[i], FALSE);
       }
     }
+
+    if(i == 2 || i == 5)
+    {
+      gtk_widget_set_halign(GTK_WIDGET(box), GTK_ALIGN_END);
+      gtk_widget_set_hexpand(GTK_WIDGET(box), TRUE);
+      gtk_container_add(GTK_CONTAINER(flow), GTK_WIDGET(box));
+      box = NULL;
+    }
+    else if(i > 2 || type != 2)
```

Comment:
type != 2
Should be enough.
Good to know about flow box! Thanks

Reply:
The time difference line (2) still has the three time fields (with ":" separator)...

Reply:
You are right. My mistake ! Thanks